ow_lander/scripts/path_planning_commander.py

```diff
@@ -33,9 +33,9 @@ def __init__(self):
     robot = moveit_commander.RobotCommander()
     scene = moveit_commander.PlanningSceneInterface()
 
-    move_arm = moveit_commander.MoveGroupCommander("arm")
-    move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    move_grinder = moveit_commander.MoveGroupCommander("grinder")
+    move_arm = moveit_commander.MoveGroupCommander("arm", wait_for_servers=15.0)
```

Comment:
This is confusing me. The wait_for_servers pieces are not in melodic-devel, which implies they were either added as a hotfix to master but not melodic-devel or they were included in master at release time and later removed from melodic-devel. If the latter, we should not include this change.

Reply:
After the merges (master -> melodic-devel -> OW-496) this diff is gone and everything looks as expected!